### DISCUSSION TITLE
NvCheatsheet - grid is not gridding?!

### DISCUSSION THREAD

**Discussion by franticflux** [NvChad/NvChad#2846] [Q&A] [answered]
So I recently updated to Neovim 0.10 and in the process I changed from NvChad 2.0 to 2.5. Everything works perfectly (after some minor tweaking) and I really do like the new structure for configs, excellent work guys! However, there is one small thing that really bothers me.  On laptops / screens with lower resolution, the grid of NvCheatsheet is not giving me grids. (Unless I make the font really really small.) Is there an easy way to alter this behaviour? I already have theme = "grid" obviously

**Comment by siduck** [comment, marked as answer]
Can u add Opens in excluded_groups?

```lua
  cheatsheet = {
    theme = "grid", -- simple/grid
    excluded_groups = { "terminal (t)", "autopairs", "Nvim", "Opens" },  
  },
```

**Reply by franticflux** [reply]
Wow, that worked thanks! But why opens, what?! urhm Uh?

I think I get it, The description is just too long for "gx". Will shorten it...

**Reply by siduck** [reply]
![image](https://github.com/NvChad/NvChad/assets/59060246/8f398b46-160c-4895-a56b-0bb39c744960)

look at the gx mapping, i'll exclude it from  nvcheatsheet. You can remove the change now, i have made it default

**Reply by benzwt** [reply]
where should 

> Can u add Opens in excluded_groups?
> 
> ```lua
>   cheatsheet = {
>     theme = "grid", -- simple/grid
>     excluded_groups = { "terminal (t)", "autopairs", "Nvim", "Opens" },  
>   },
> ```

which file I should paste these lines to ?

**Reply by siduck** [reply]
@benzwt  read :h nvui to know the options config! u should put it in root table of your chadrc
